.env.example

```diff
@@ -9,5 +9,4 @@
 ROLLBAR_ACCESS_TOKEN=ROLLBAR_ACCESS_TOKEN
 ROLLBAR_ENV=development
 
-# TODO: Replace `rails-template` with the name of the app.
-DATABASE_URL=postgres://postgres@localhost:5432/rails-template-development
+DATABASE_URL=postgres://postgres@localhost:5432/roda-development
```

Comment:
A benefit of rebasing this branch on top of develop (rather than merging) means that this other unrelated work will not show in this PR. Making self contained and easier to review. 

I think we can fix this by:

1.  doing an interactive rebase and deleting this merge commit `f2dbe75d5b6069f3f91628d49b76d0c8dc39be94`
2. rebasing this new work on top of the existing code on develop `git rebase origin/develop`
3. updating this pull request with the new history `git push --force-with-lease`